WORKSPACE.yaml

```diff
@@ -7,8 +7,8 @@ defaultArgs:
   publishToNPM: true
   publishToJBMarketplace: true
   localAppVersion: unknown
-  codeCommit: b3e87fd56f3d781f19865c315ac4f372bf5fd31a
-  codeVersion: 1.76.0
+  codeCommit: eedb6867a25066a77416d154c60d2d73869326c2
```

Comment:
This is me learning 👋🏼 Where do we get this commit?

Reply:
@laushinka this is precisely the commit you get from VS Code's source code: 
<img width="1395" alt="image" src="https://user-images.githubusercontent.com/29888641/224349903-47e8a3ed-a071-4b88-a5a8-ee545f1928b8.png">